app/src/main/res/values-night/color_palette.xml

```diff
@@ -231,5 +231,10 @@
   <!-- ON-BOARDING -->
   <color name="color_palette_onboarding_primary_color">@color/color_def_dark_green</color>
   <color name="color_palette_onboarding_primary_text_color">@color/color_def_accessible_grey</color>
-  <color name="color_palette_onboarding_profile_type_background_color">@color/color_def_dark_jade</color>
+  <color name="color_palette_onboarding_profile_type_background_color">@color/color_def_jade</color>
+  <color name="color_palette_onboarding_edit_icon_color">@color/color_def_oppia_green</color>
+  <color name="color_palette_onboarding_black_color">@color/color_def_black</color>
+
+  <color name="color_palette_text_error_color">@color/color_def_oppia_reddish_brown</color>
```

Comment:
I created anew one because the actual color values are different. The existing one is pink which is pale on a white background and may not pass accessibility scanner checks, so I created a darker version.